IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedMgmtElements/PropertySets/Pset_ProjectOrderMaintenanceWorkOrder/DocPropertySet.xml

```diff
@@ -8,7 +8,6 @@
 		<DocProperty xsi:nil="true" href="WorkTypeRequested_1S5uo0qVSHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="ContractualType_1b25u0qVSHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="IfNotAccomplished_1kaSO0qVSHuO00025QrEV" />
-		<DocProperty xsi:nil="true" href="MaintenaceType_1u6ou0qVSHuO00025QrEV" />
```

Comment:
please undo this change i will log it as a Bug to be updated

Reply:
@czapplitec has done this change. What do you mean by undo ?

Reply:
Some one removed an existing property that has a spelling mistake this is out of scope for this work so should be reverted

Reply:
Done. commit : aaa9761287fa708fc2fecb7187beac2320ebdcce

Reply:
woops... see latest comment!